scripts/street_sign_recognizer.py

```diff
@@ -17,14 +18,63 @@ def __init__(self):
         """ Initialize the street sign reocgnizer """
         rospy.init_node('street_sign_recognizer')
         self.cv_image = None                        # the latest image from the camera
+        self.hsv_image = None                       # converted hsv image
+        self.mask = None
         self.bridge = CvBridge()                    # used to convert ROS messages to OpenCV
         cv2.namedWindow('video_window')
         rospy.Subscriber("/camera/image_raw", Image, self.process_image)
+        
+        cv2.namedWindow('threshold_image')
+        self.hsv_lb = np.array([0, 0, 0]) # hsv lower bound
+        cv2.createTrackbar('H lb', 'threshold_image', 0, 255, self.set_h_lb)
```

Comment:
sliders are nice, but it's also nice to enter a default value so you don't have to re-tune it every time you run the code